contracts/ProfileToken.sol

```diff
@@ -15,18 +15,27 @@ contract ProfileToken is
 
     Counters.Counter public _tokenIds;
 
+    mapping (address => bool) public tokenExists;
+
     event NewProfile(uint256 tokenId, address owner);
 
     constructor() {
     }
 
+    //check if connected wallet-address already has a tokenID
+    function checkEntry(address _address) external pure returns(bool) {
+        return tokenExists[_address];
+    }
```

Comment:
I think we can call the ERC721 balanceOf(address) instead. If the return value is greater than 0, then you know the address has ownership of 1 token or more.